chat-commands.js

```diff
@@ -513,7 +514,7 @@ exports.commands = {
 		targetUser.lastPM = user.userid;
 		user.lastPM = targetUser.userid;
 	},
-	pminfoboxhelp: [`/pminfobox [user], [html]- PMs an [html] infobox to [user]. Requires * ~`],
+	pminfoboxhelp: [`/pminfobox [user], [html]- PMs an [html] infobox to [user]. Requires * ~.`],
```

Comment:
We've never used `.` in the "Requires" list, because it could be mistaken for a rank symbol.

Reply:
Oh